### PR TITLE
[FrameworkBundle] Update serializer configuration reference

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.7 |
| Fixed tickets | n/a |

### CODE REVIEW

**Comment by xabbuh** [comment]
What do you think about including these changes in #5335 so that we can just link to the related cookbook entries?

**Comment by dunglas** [comment]
Rebased and added links to new cookbook entries.

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .. index::
-    single: Configuration reference; Framework
+single: Configuration reference; Framework
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: This needs to be removed Otherwise this line is not treated as an argument for the index.

Comment (WouterJ) [inline reference/configuration/framework.rst]: same applies to other directives below

**Comment by dunglas** [comment]
Should be OK now.

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Looks perfect to me. Thanks @dunglas for keep on top of all of these changes!

**Pull request merged**